submissions/Shrai-dev/js_dom/main.js

```diff
@@ -0,0 +1,105 @@
+  },
+  {
+    id: 2,
+    navTitle: "Place 2",
+    title: "Gothic Quarter",
+    src: "./assets/place2.jpg",
+    text:
+      "For 2,000 years, the Gothic Quarter has been the spiritual and secular center of the city. Relics of ancient Roman buildings are still found here, but the Middle Ages are best represented by the historic monuments packed into this quarter. A masterpiece of Gothic architecture, the medieval cathedral stands on Monte Tabor, the highest point in the town center. The Gothic Quarter is where Christopher Columbus was received by the Catholic Monarchs after his first voyage to the New World, and since the 14th and 15th centuries, the city administrations have had their seat here.",
+  },
+  {
+    id: 3,
+    navTitle: "Place 3",
+    title: "The Magic Fountain",
+    src: "./assets/place3.jpg",
+    text:
+      "One of the favorite things to do in Barcelona at night is to watch the Magic Fountain of Montjuïc, at the beginning of Avinguda Maria Cristina in the Montjuïc neighborhood. The large Art Deco fountain, erected in 1929, delights all ages with its light and water shows choreographed to music. It was designed by Carles Buigas for the 1929 International Exhibition, which took place in Montjuïc. The show lasts for about an hour.",
+  },
+  {
+    id: 4,
+    navTitle: "Place 4",
+    title: "Parc Güell: Gaudí's Surrealist Park",
+    src: "./assets/place4.jpg",
+    text:
+      "Colorful, cheerful, and full of whimsy, this splendid surrealistic park is another UNESCO World Heritage Site designed by Antoni Gaudí. Created between 1900 and 1914, the Park Güell is beautifully landscaped and features architectural elements in Gaudí's signature style. Viaducts, grottoes, a colonnaded hall, winding staircases, and semi-closed conversation seats are scattered throughout the space. These creative structures are decorated in multicolored ceramic fragments. A spectacular terrace offers panoramic views of the city and the sea. Gaudí himself loved this area of the city, and his home was located here.",
+  },
+  {
+    id: 5,
+    navTitle: "Place 5",
+    title: "Casa Batlló",
+    src: "./assets/place5.jpg",
+    text:
+      "Yet another amazing Gaudí creation, the UNESCO-listed Casa Batlló is one of the most characteristic Modernist buildings in Barcelona. The fantastical mansion was designed as a private residence for the textile manufacturer Josep Batlló i Casanovas. With its freely swinging shapes and ornamental facade, this dreamlike building looks like a castle from a surreal fairy tale.",
+  },
+];
+const header = document.querySelector(".header");
+
+const headerTitle = document.createElement("h1");
+headerTitle.className = "header__title";
```

Comment:
Use [classList](https://developer.mozilla.org/ru/docs/Web/API/Element/classList) API for manipulating with classes.

Reply:
Thanks for your comments! Everything is fixed.

Reply:
Not fixed yet. [Read more](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) how to add class to the element using classList.

Reply:
Done)